spec/03-plugins/31-proxy-cache/02-access_spec.lua

```diff
@@ -1179,6 +1179,53 @@ do
         assert.same("Hit", res.headers["X-Cache-Status"])
       end)
 
+     it("response status with 304 empty Content-Type", function()
+        local res = assert(client:send {
+          method = "GET",
+          path = "/status/304",
+          headers = {
+            host = "route-10.com",
+          },
+        })
+
+        assert.res_status(304, res)
+        assert.same("Miss", res.headers["X-Cache-Status"])
+
+        res = assert(client:send {
+          method = "GET",
+          path = "/status/304",
+          headers = {
+            host = "route-10.com",
+          },
+        })
+
+        assert.res_status(304, res)
+        assert.same("Hit", res.headers["X-Cache-Status"])
```

Comment:
This test is not actually testing the change made. Probably it should be:

```suggestion
        assert.same("Hit", res.headers["X-Cache-Status"])
        assert.is_nil(res.headers["Content-Type"])
```

Reply:
Without the patch this test fails, with the test passed. We want to test that 304 are put in cache not that the endpoint is respecting the RFC by no setting the content type

Reply:
I update the mock to no send data on 204 and 304